apps/omg_api/lib/root_chain_coordinator/core.ex

```diff
@@ -108,8 +108,26 @@ defmodule OMG.API.RootChainCoordinator.Core do
   @doc """
   Gets synchronized height
   """
-  @spec get_synced_height(t()) :: {:sync, non_neg_integer()} | :nosync
-  def get_synced_height(state) do
+  @spec get_synced_height(t(), atom() | pid()) :: {:sync, non_neg_integer()} | :nosync
+  def get_synced_height(state, pid) when is_pid(pid) do
```

Comment:
what happens if one service will be restarts?

Reply:
We talk about, it should be fine as with every `check_in` call `pid` + `service_name` is passed.